frontend/packages/lib-components/src/atoms/Chip.tsx

```diff
@@ -44,26 +44,34 @@ export const SelectionChip = React.memo(function SelectionChip({
   selected,
   onChange
 }: SelectionChipProps) {
+  const ariaId = Math.random().toString(36).substring(2, 15)
```

Comment:
This generates a new id on every render...seems unnecessary?
IMHO it would be also nicer to separate the intent (= generate a unique ID) from the implementation. We don't even need to generate random numbers, because a global incrementing id with a string prefix works fine in frontend code.

For example:

```typescript
let id = 0
export const uniqueId = () => id++
```

```typescript
const ariaId = useEffect(() => `aria-${uniqueId()}`, [])
```

Reply:
I'll try that out! Or maybe some useAriaId hook? cc: @massproduction

Reply:
Hmm, since we want to start QA-testing now and this is already used in other places, I'll just add a ticket on tech backlog for now.